src/platforms/apple/common/install/swift-package-manager.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Swift Package Manager (SPM)
+---
+
+To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Swift Packages` -> `Add Package Dependency`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git`, and select a version (or branch) on the next page.
+
+<Note><markdown>
```

Comment:
I had this problem trying to use Sentry: https://github.com/bruno-garcia/SentryCocoaLumberjack/blob/e30623b76a27647e87d8cba0752724b782ec40ab/Package.swift#L13

can we replace `version` below with the latest from release registry magically here?

```suggestion

When using SPM version 5.2 or later, you must specify the the target name too:

```swift
.package(name: "Sentry", url: "https://github.com/getsentry/sentry-cocoa", from: "version"),
```

<Note><markdown>
```